mixer/adapter/redisquota/redisquota.go

```diff
@@ -237,6 +241,8 @@ func (b *builder) Build(context context.Context, env adapter.Env) (adapter.Handl
 		dimensionHash: dimensionHash,
 	}
 
+	env.Logger().Infof("Redis Quota Adapter Built")
```

Comment:
We don't really need this logging, Mixer is already handling this.

Reply:
Thanks.. Removed